std/file.d

```diff
@@ -3246,6 +3246,58 @@ if ((isSomeFiniteCharInputRange!RO || isConvertibleToString!RO) &&
     }
 }
 
+version(Windows)
+{
+    import core.sys.windows.w32api:_WIN32_WINNT;
+    static if(_WIN32_WINNT >= 0x600) //WindowsVista or later
+    {
+        /** 
+		 * 
+		 * Params:
+		 *   original = The original path where the link will redirect
+		 *   link = The path where the link will be created
+		 */
+        void symlink(string original, string link)
+        {
+            import core.stdc.string:strlen;
+            import core.sys.windows.winbase;
+            import core.sys.windows.windef:DWORD, MAX_PATH, LPWSTR;
+            import std.utf:toUTF16z;
+            import std.file:FileException;
+    
+            DWORD typeFlag = 0; //File
+            if (std.file.isDir(original))
+                typeFlag = SYMBOLIC_LINK_FLAG_DIRECTORY;
+            typeFlag|= SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE;
```

Comment:
This requires that "Developer mode" is enabled on the machine, right? That should probably be documented.